content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/fefo.rst

```diff
@@ -0,0 +1,123 @@
+   16th** (Jan 31 - 15).
+
+.. _inventory/removal/exp-date:
+
+To view the expiration dates of items in stock, navigate to the product form and click the
+:guilabel:`On Hand` smart button.
+
+Next, click the additional options icon on the far right and select the columns
+:guilabel:`Expiration Date` and :guilabel:`Removal Date`.
+
+.. image:: fefo/removal-date.png
+   :align: center
+   :alt: Show expiration dates from the inventory adjustments model accessed from the *On Hand*
+         smart button from the product form.
+
+Workflow
+--------
+
+Using the :abbr:`FEFO (First Expired, First Out)` removal strategy ensures that products with the
+nearest removal date are picked first.
+
+To understand how this removal strategy works, consider the following example below about the
+product, `Egg carton`, which is a box containing twelve eggs. The product is tracked :guilabel:`By
+Lots`, and the product category's :guilabel:`Force Removal Strategy` is set to :guilabel:`First
+Expired, First Out (FEFO)`
+
+.. seealso::
+   - :ref:`Set up force removal strategy <inventory/routes/strategies/set>`
+   - :ref:`Enable lots tracking <inventory/removal/lots-setup>`
+   - `Odoo Tutorials: Perishable Products <https://www.youtube.com/watch?v=8zAlNcdg0ig>`_
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * -
+     - LOT1
+     - LOT2
+     - LOT3
+   * - On-hand stock
+     - 5
+     - 2
+     - 1
+   * - Expiration date
+     - April 4
+     - April 10
+     - April 15
+   * - :ref:`Removal date <inventory/removal/exp-date>`
+     - February 26
+     - March 4
+     - March 9
+
+To see the removal strategy in action, create a :ref:`delivery order <inventory/delivery/one-step>`
+for six cartons of eggs, either by going to the :menuselection:`Sales app` and creating a new
+quotation, or from the delivery orders dashboard in :menuselection:`Inventory --> Operations -->
+Deliveries`. :guilabel:`Confirm` the sales order or click :guilabel:`Mark as Todo` on the draft
+delivery order.
+
+Doing so creates the delivery order for today, December 29th, and the lot numbers with the soonest
+expiration dates are reserved thanks to the :abbr:`FEFO (First Expired, First Out)` removal
+strategy.
+
+To view the detailed pickings, click the :guilabel:`⦙≣ (bulleted list)` icon on the far right of the
+egg's product line in the :guilabel:`Operations` tab of the delivery order. Doing so opens the
+:guilabel:`Open: Stock move` pop-up window.
+
+In the :guilabel:`Open: Stock move` window, the :guilabel:`Pick from` field details where the
+quantities to fulfill the :guilabel:`Demand` are picked from. Since the order demanded six cartons
+of eggs, using the :abbr:`FEFO (First Expired, First Out)` removal strategy, all five cartons from
+`LOT1` with the removal date of February 26th are picked. The remaining carton is selected from the
+`LOT2`, which has a removal date of March 4.
```

Comment:
```suggestion
In the :guilabel:`Open: Stock move` pop-up window, the :guilabel:`Pick from` field display where the
quantities to fulfill the :guilabel:`Demand` are picked from.

Since the order demanded six Cartons of eggs, using the :abbr:`FEFO (First Expired, First Out)`
removal strategy, all five Cartons from `LOT1`, with the removal date of February 26th are picked.
The remaining Carton is selected from `LOT2`, which has a removal date of March 4th.
```